lib/host/gen_helpers.js

```diff
@@ -166,7 +166,17 @@ const genHelpers = (state = {}, adapter) => {
       } = orders
 
       // NOTE: No await, in order to update cancelled order set immediately
-      adapter.cancelOrder(connection, order)
+      adapter.cancelOrder(connection, order).then((order) => { // eslint-disable-line
+        debug(
+          'order successfully cancelled (id: %s, cid: %s): %s %f @ %f',
+          order.id, order.cid, order.type, order.amountOrig, order.price
+        )
+      }).catch((notification) => {
+        debug(
+          'received error while canceling order(id: %s, cid: %s): %s %f @ %f [ERROR: %O]',
+          order.id, order.cid, order.type, order.amountOrig, order.price, notification
+        )
+      })
```

Comment:
1. open a twap order, mid price, slice interval 30secs, slice amount 0.01
2. in the interval, do a partial fill with a market order on 0.01
3. wait for the 30 seconds to end and the twap wants to cancel the order

error:

```
Unhandled rejection: Notification {
  _events: [Object: null prototype] {},
  _eventsCount: 0,
  _maxListeners: undefined,
  _fields: [Object],
  _boolFields: [],
  mts: 1616753628525,
  type: 'oc-req',
  messageID: null,
  notifyInfo: [Array],
  code: null,
  status: 'ERROR',
  text: 'Order not found.',
  [Symbol(kCapture)]: false
}
Unhandled rejection: Notification {
  _events: [Object: null prototype] {},
  _eventsCount: 0,
  _maxListeners: undefined,
  _fields: [Object],
  _boolFields: [],
  mts: 1616753628525,
  type: 'oc-req',
  messageID: null,
  notifyInfo: [Array],
  code: null,
  status: 'ERROR',
  text: 'Order not found.',
  [Symbol(kCapture)]: false
}
! process.<anonymous> -> process (/Users/robert/bitfinex/bfx-hf-ui/node_modules/bfx-hf-util/lib/catch_uncaught_errors.js) [41:%s]
! emit -> process (events.js) [326:%s]
! processPromiseRejections -> null (internal/process/promises.js) [209:%s]
! processTicksAndRejections -> process (internal/process/task_queues.js) [98:%s]
! process.<anonymous> -> process (/Users/robert/bitfinex/bfx-hf-server/node_modules/bfx-hf-util/lib/catch_uncaught_errors.js) [41:%s]
! emit -> process (events.js) [326:%s]
! processPromiseRejections -> null (internal/process/promises.js) [209:%s]
! processTicksAndRejections -> process (internal/process/task_queues.js) [98:%s]

source from /Users/robert/bitfinex/bfx-hf-ui/node_modules/bfx-hf-util/lib/catch_uncaught_errors.js:41
  })
}

process.on('unhandledRejection', (reason) => {
  logger('Unhandled rejection: %s', reason instanceof Error ? reason.message : reason)
!!!
  onError(reason instanceof Error ? reason : new Error(reason))
!!!
}).on('uncaughtException', err => {
  logger('Uncaught exception: %s', err.message)
  onError(err)
})

source from /Users/robert/bitfinex/bfx-hf-server/node_modules/bfx-hf-util/lib/catch_uncaught_errors.js:41
  })
}

process.on('unhandledRejection', (reason) => {
  logger('Unhandled rejection: %s', reason instanceof Error ? reason.message : reason)
!!!
  onError(reason instanceof Error ? reason : new Error(reason))
!!!
}).on('uncaughtException', err => {
  logger('Uncaught exception: %s', err.message)
  onError(err)
})
```